lib/std/os/uefi/protocols/device_path_protocol.zig

```diff
@@ -454,9 +454,7 @@ pub const MessagingDevicePath = union(Subtype) {
             console_protocol: bool,
             storage_protocol: bool,
             network_protocol: bool,
-
-            // u1 + 4 * bool = 5 bits, we need a total of 32 bits
-            reserved: u27,
```

Comment:
This should still be u27, the mistake was that the ControllerType should be `enum(u1)`